client/components/telecom/alias/hunting/sounds/telecom-telephony-alias-hunting-sounds.html

```diff
@@ -13,55 +13,55 @@
     </div>
 
     <div class="modal-body clearfix">
-
         <h3 class="modal-title"
             data-translate="telephony_alias_hunting_sounds_manager">
         </h3>
 
-        <input-file data-ng-model="$ctrl.toUpload"
-                    data-ng-accept="audio/*"
-                    data-ng-accept-filter="$ctrl.checkValidAudioExtention(file)">
-            <i class="fa fa-upload mr-1"></i>
-            <span data-translate="telephony_alias_hunting_sounds_upload"></span>
-        </input-file>
+        <div class="form-group mb-5">
```

Comment:
You can remove `mb-5`.

Reply:
It's here for add space when validation button of selected file is shown